packages/smithy/smithy_codegen/pubspec.yaml

```diff
@@ -9,8 +9,8 @@ environment:
 
 dependencies:
   args: ^2.3.0
-  aws_common: ">=0.5.0 <0.6.0"
-  aws_signature_v4: ">=0.4.0 <0.5.0"
+  aws_common: any
+  aws_signature_v4: any
```

Comment:
not sure what is this change for?

Reply:
It's from this: https://github.com/aws-amplify/amplify-flutter/pull/3568